client/src/model/slices/map_data_slice.ts

```diff
@@ -56,27 +55,49 @@ const mapSlice = createSlice({
 export const setCurrentDataLayer = (layerId: string) => {
   return async (dispatch: AppDispatch) => {
     if (layerId == null) {
-      dispatch(setCurrentDataLayerError({ error: 'Invalid map state' }));
+      dispatch(setMapDataError({ error: 'Invalid map state' }));
     } else {
-      dispatch(setCurrentDataLayerSuccess({ currentDataLayerId: layerId as MapLayer }));
+      dispatch(setMapDataSuccess({ currentDataLayerId: layerId as MapLayer }));
     }
   };
 };
 
 /**
- * Change the zoom level.
+ * Update the integer zoom level.
  */
 export const setZoom = (zoom: number) => {
   return async (dispatch: AppDispatch) => {
     if (zoom == null) {
-      dispatch(setCurrentDataLayerError({ error: 'Invalid zoom' }));
+      dispatch(setMapDataError({ error: 'Invalid zoom' }));
     } else {
-      dispatch(setCurrentDataLayerSuccess({ zoom: zoom }));
+      dispatch(setMapDataSuccess({ zoom: zoom }));
     }
   };
 };
 
+/**
+ * Change the zoom to either an address, water system, or zip code level.
+ */
+export const setZoomLevel = (level: ZoomLevel) => {
+  return async (dispatch: AppDispatch) => {
```

Comment:
What does the `async` keyword do here if there's nothing being `await`ed within the function?

Reply:
nothing! copied / pasted and forgot to delete. Thanks!